dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/parameters/AbstractParameters.java

```diff
@@ -17,23 +17,22 @@
 
 package org.apache.dolphinscheduler.plugin.task.api.parameters;
 
+import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
Please remove unnessnary change in this file.

Reply:
Already processed, code already committed